open-metadata-resources/open-metadata-deployment/charts/lab/README.md

```diff
@@ -138,16 +137,36 @@ by deleting the deployment and running the installation again. This will wipe ou
 metadata across the lab Egeria servers, remove all messages from the Kafka bus used in the cohort,
 reset the Jupyter notebooks to their original clean state, etc.
 
-To delete the deployment, simply run:
+To delete the deployment, simply run this for Helm3:
 
 ```shell script
-$ helm delete labs --purge
+$ helm delete lab
+```
+Or if using Helm2:
+```$xslt
+helm delete lab --purge
 ```
 
-Where `labs` is the name you used in your original deployment. (If you left Helm to randomly
-generate one, you can see what it's called by first running `helm list` and reviewing the output.)
+Where `lab` is the name you used in your original deployment. (You can see what it's called by first running `helm list` and reviewing the output.)
 
 (Then just re-run the last command in the Installation section above to get a fresh environment.)
+## Port Clashes
+The chart is configured to use a fixed set of ports and expose them using a 'NodePort' service as described above.
+
+You may find you clash with other services setup in your cluster. If so you can override the ports by creating a file such as 'lab.yaml' with the following contents:
+```
+service:
+  type: NodePort
+  nodeport:
+    jupyter: 30888
+    core: 30080
+    datalake: 30081
+    dev: 30082
+    ui: 30443
+```
+and then change the port numbers accordingly.
+You can then deploy using
+'helm install lab lab -f lab.yaml' which will override standard defaults with your choices
```

Comment:
Suggest using back-ticks rather than single quotes here:

'lab.yaml' -> `lab.yaml`
'helm install lab lab -f lab.yaml' -> `helm install lab lab -f lab.yaml`

Reply:
Agree. done. thanks